keylime-push-model-agent/Cargo.toml

```diff
@@ -0,0 +1,40 @@
+[package]
+name = "keylime_push_model_agent"
+description = "Rust agent for Keylime (Push Model)"
+authors.workspace = true
+edition.workspace = true
+license.workspace = true
+repository.workspace = true
+version.workspace = true
+
+[dependencies]
+actix-web.workspace = true
```

Comment:
If I understand it correctly, the prototype only needs the `actix-web` crate for testing. We need to find a way to make it optional when compiling, only required when testing. It doesn't need to be done as part of this PR.

Reply:
Sorry, but I don't understand it very well ... why is it a problem taking into account it is being used in other parts of the project? I would understand if it was a brand new dependency just being included for this ... but in this case ... I don't find the reason (I might be wrong in any case)

Reply:
The main reason I see is for clarity: setting it as optional only for testing shows that the final release binary do not depend on the crate.

A minor reason would be to speed-up the build in case someone wants to build only this individual package without tests, and not all the packages in the workspace (TBH, I don't know if that is actually achievable)

As I wrote before, this change is a nice to have, you don't need to change it for now.